components/screens/Home.tsx

```diff
@@ -2,12 +2,12 @@ import React, { useState, useEffect, FunctionComponent } from 'react'
 
 import { Button, Image, TouchableOpacity, StyleSheet, Text, View,} from 'react-native'
 
-import firestore from '@react-native-firebase/firestore'
+// import firestore from '@react-native-firebase/firestore'
 
-import { t } from 'react-native-tailwindcss';
-import styled from 'styled-components/native';
-import { StackNavigationProp } from '@react-navigation/stack';
-import { RootStackParamList } from '../../App';
+import { t } from 'react-native-tailwindcss'
```

Comment:
Just a heads up - will be refactoring tailwind so we can use nativewind instead.

Reply:
👍